### ISSUE TITLE
USER bioc incompatible with s6 layer

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/bioconductor#9]
E.g., see: https://github.com/janeczku/docker-alpine-kubernetes/issues/7

Trying to initiate rstudio-server produces an error like:
```
s6-mkdir: warning: unable to mkdir /var/run/s6: Permission denied
```

I think tinytex can be installed as root, will try.

**Comment by LiNk-NY** [comment]
I think this is the root of the issue: 
https://github.com/lablup/backend.ai-agent/issues/103

**Comment by lwaldron** [comment]
Addressed in 847ff75, with tinytex installed to `/usr/local/lib` as root. Also adds `chmod -R a+w` on library directories. At the moment I can't see anything wrong with this for the use cases I have in mind, but let me know if you do.  Will close for now.
